codegenerator/cli/templates/static/codegen/src/ink/components/CustomHooks.res

```diff
@@ -53,8 +53,8 @@ module InitApi = {
     | @as("white") White
     | @as("gray") Gray
 
-  let toTheme = (kind: messageKind): Style.chalkTheme =>
-    switch kind {
+  let toTheme = (color: messageColor): Style.chalkTheme =>
+    switch color {
```

Comment:
Does `:>` work here?

Reply:
They are not the same underlying string though...

Reply:
Ah, different casing or something